upgrade-2.x.sh

```diff
@@ -895,6 +896,15 @@ else
     log "No supervisor updater fix is required..."
 fi
 
+# Fix issue with `read` on 2.10.x/2.11.0 resinOS versions
+if version_gt "$VERSION_ID" "2.9.7"; then
```

Comment:
We should avoid applying for versions which will have it fixed.

Reply:
@agherzan changed, and that was the plan, just didn't know if it will be fixed.